pkg/ddc/juicefs/data_migrate.go

```diff
@@ -214,6 +211,33 @@ func (j *JuiceFSEngine) generateDataMigrateValueFile(r cruntime.ReconcileRequest
 	return valueFile.Name(), nil
 }
 
+func (j *JuiceFSEngine) setParallelMigrateOptions(dataMigrateInfo *cdatamigrate.DataMigrateInfo, dataMigrate *datav1alpha1.DataMigrate) {
+	var err error
+	dataMigrateInfo.ParallelOptions = cdatamigrate.ParallelOptions{
+		SSHPort:                cdatamigrate.DefaultSSHPort,
+		SSHReadyTimeoutSeconds: cdatamigrate.DefaultSSHReadyTimeoutSeconds,
+		SSHSecretName:          dataMigrate.Spec.ParallelOptions[cdatamigrate.SSHSecretName],
+	}
+
+	sshPort, exist := dataMigrate.Spec.ParallelOptions[cdatamigrate.SSHPort]
+	if exist {
+		dataMigrateInfo.ParallelOptions.SSHPort, err = strconv.Atoi(sshPort)
+		if err != nil {
+			j.Log.Info("sshReadyTimeoutSeconds in the parallelOptions is not a int, use default value.")
```

Comment:
How about return err and fail fast? Ignoring the error may let users have incorrect expectation?

Reply:
thanks, fixed.